recipes/scvi/meta.yaml

```diff
@@ -1,47 +1,29 @@
 {% set name = "scvi" %}
-{% set version = "0.4.1" %}
+{% set version = "0.5.0" %}
 
 package:
   name: '{{ name|lower }}'
   version: '{{ version }}'
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 23cab6850f433e85aa23c12ddfe7fb74e12b481d584e5f519e5f51f60d6e64cf
+  sha256: ea56364ce84c2dd601735d465aea04cefdcb709311bb3892cd5cab02d1c151a2
 
 build:
   noarch: python
-  number: 1
+  number: 0
   script: python -m pip install --no-deps --ignore-installed .
 
 requirements:
   host:
+    - python >=3.7
+    - pip
+
     - pytest-runner >=4.0
-    - python >=3.6
+    - python >=3.7
```

Comment:
Does this not work with python 3.6? Please remove the duplicate `python >=3.7` line.

Reply:
No we do not ensure compatibility with python 3.6